common-java-root/gradlew.bat

```diff
@@ -0,0 +1,104 @@
+@rem
```

Comment:
Does this file need to be checked in?

Reply:
Probably not...I checked in because all the existing root projects had them checked in...going to leave it checked in since I've put it in the PR